src/containers/Home.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { fetchPetitions, clearPetitions } from 'actions/PetitionsActions';
+import Home from 'components/Home';
+import getPetitions from 'selectors/petitions';
+import getPetitionsHomeQuery from 'selectors/petitionsHomeQuery';
+import isClientSideRouting from 'helpers/isClientSideRouting';
+
+const HOME_QUERY = getPetitionsHomeQuery({ limit: 3 });
+
+const HomeContainer = withRouter(React.createClass({
```

Comment:
I don't think for this, using `withRouter` is needed, that's just when we are grabbing where params on the client-side (for filtering or by `:id`)